src/coreimage.cs

```diff
@@ -2645,29 +2788,23 @@ interface CIImageProcessorKernel {
 	[iOS (8,0)]
 	[Mac (10,10)]
 	[BaseType (typeof (CIFilter))]
-	interface CIAccordionFoldTransition {
+	interface CIAccordionFoldTransition : ICIAccordionFoldTransitionProtocol {
```

Comment:
@spouliot I see you are using the `I` version of `CIAccordionFoldTransitionProtocol` is that intentional? IIRC you have to use the `I` less version when doing this else the generator won't do the right thing.

Also I see this being mixed in the whole PR, for example the following does what I expect :)

`interface CIAccordionFoldTransitionProtocol : CITransitionFilterProtocol {`

Reply:
good catch, I'll review those

`CIFilter` subclasses are generated differently that _normal_ `NSObject`so the end-result might be (nearly) identical. IOW I think my generator changes are not correct, with incorrect data giving the correct output.